pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -809,6 +809,23 @@ const (
 	// for administrators.
 	ClusterRoleNameGardenerAdministrators = "gardener.cloud:system:administrators"
 
+	// GardenerReadOnlyClusterRoleName is the name of a cluster role allowing read-only access to a shoot cluster.
+	GardenerReadOnlyClusterRoleName = "gardener.cloud:system:read-only"
+	// GardenerSystemAdminsGroupName is a group assigned to gardener system administrators
+	// when they are using the AdminKubeconfig to access a shoot cluster.
+	GardenerSystemAdminsGroupName = "gardener.cloud:system:admins"
+	// GardenerSystemViewersGroupName is a group assigned to gardener system viewers
+	// when they are using the ViewerKubeconfig to access a shoot cluster.
+	GardenerSystemViewersGroupName = "gardener.cloud:system:viewers"
+	// GardenerProjectAdminsGroupName is a group assigned to gardener project administrators,
+	// or other users allowed to create shoot/adminkubeconfig on the shoot but not system administrator,
+	// when they are using the AdminKubeconfig to access a shoot cluster.
+	GardenerProjectAdminsGroupName = "gardener.cloud:project:admins"
```

Comment:
I would rephrase this as something similar to

GardenerProjectAdminsGroupName is a group assigned during AdminKubeconfig generation to gardener project administrators or other users allowed to request an AdminKubeconfig. System administrators do not get assigned to this group when requesting an AdminKubeconfig. Instead, they are assigned to the group "gardener.cloud:system:admins".